src/calculateRentalCost.js

```diff
@@ -28,7 +28,15 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  if (days >= 7) {
+    return days * 40 - 50;
```

Comment:
Good solution. To make it more adjastable you could put 40 into variable. So when the car cost for a day changes you can modify it in just one place